luajwtjitsi.lua

```diff
@@ -93,7 +93,10 @@ function M.encode(data, key, alg)
 		return nil, "Algorithm not supported"
 	end
 
-	local header = { typ='JWT', alg=alg }
+	header = header or {}
+
+	header['typ']='JWT'
```

Comment:
just a little detail, but everywhere in the code there is a space before and after very '=' sign